configure.ac

```diff
@@ -6044,6 +6044,17 @@ then
     AM_CFLAGS="$AM_CFLAGS -DHAVE_XCHACHA"
 fi
 
+# ASCON
```

Comment:
Should it be part of experimental or all yet?

Reply:
Its a complete but limited implementation. Its fair to list it under experimental for now. Should I require `WOLFSSL_EXPERIMENTAL_SETTINGS` to enable ASCON for now?

Reply:
Yes let's put ASCON under experimental for now. Thank you

Reply:
Added.